stix_shifter_modules/secretserver/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,78 @@
+{
+  "username": [
+    {
+    "key": "user-account.user_id"
+  },
+    {
+      "key": "x-ibm-finding.user_name",
+       "object": "finding"
+   }
+  ],
+
+  "IpAddress": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "src_ip"
+    },
+    {
+      "key": "network-traffic.src_ref",
+      "object": "nt",
+      "references": "src_ip"
+    }
+  ],
+   "Server": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "dst_ip"
+    },
+    {
+      "key": "network-traffic.dst_ref",
+      "object": "nt",
+      "references": "dst_ip"
+    }
+  ],
+
+
+  "StartTime": [
+    {
+      "key": "first_observed",
+      "transformer": "EpochToTimestamp",
+      "cybox": false
+    },
+    {
+      "key": "last_observed",
+      "transformer": "EpochToTimestamp",
+      "cybox": false
+    }
+  ],
+  "Url": {
+    "key": "url.value"
+  },
+
+  "EventSubject" :
+    {
+      "key": "x-ibm-finding.name",
+      "object": "finding"
+    },
+  "EventNote":
+  {
+     "key": "x-ibm-finding.description",
+      "object": "finding"
+  },
+  "EventDetails":
+  {
+     "key": "x-ibm-finding.finding_type",
+      "object": "finding"
+  },
+  "EventTime":
+  {
+     "key": "x-ibm-finding.event_occurrence",
```

Comment:
This should be `x-ibm-finding. time_observed`, just make sure it's converted into a UTC timestamp if it's not coming in as that already.